bundle/Resources/config/services/commands.yaml

```diff
@@ -53,3 +53,12 @@ services:
             - "@?profiler"
         tags:
             - { name: console.command, command: 'ngsite:profiler:clear-cache' }
+
+    ngsite.command.tag_content:
+        class: Netgen\Bundle\SiteBundle\Command\TagContentByTypesCommand
+        arguments:
+            - "@ibexa.api.repository"
+            - "@netgen_tags.api.service.tags"
+            - "@ibexa.config.resolver"
+        tags:
+            - { name: console.command, command: 'ngsite:content:tag-content' }
```

Comment:
Missing empty line at the end of the file.

Reply:
fixed in 8d1482b9